app/bundles/PointBundle/Form/Type/GenericPointSettingsType.php

```diff
@@ -1,46 +0,0 @@
-<?php
-
-namespace Mautic\PointBundle\Form\Type;
-
-use Symfony\Component\Form\AbstractType;
-use Symfony\Component\Form\Extension\Core\Type\NumberType;
-use Symfony\Component\Form\FormBuilderInterface;
-use Symfony\Component\Validator\Constraints\NotEqualTo;
-
-/**
- * @extends AbstractType<array<mixed>>
- */
-class GenericPointSettingsType extends AbstractType
```

Comment:
We cannot remove classes in a minor release. It will have to be removed in a major release (Mautic 6.0.0). Please, revert this class and mark it as deprecated.

Reply:
Done